src/SamplesApp/UITests.Shared/Windows_Storage/StorageFileOperationsTests.xaml.cs

```diff
@@ -30,6 +31,7 @@ private void StorageFileOperationsTests_DataContextChanged(Windows.UI.Xaml.Depen
 	public class StorageFileOperationsTestsViewModel : ViewModelBase
```

Comment:
This should be in a separate file.

Reply:
I put the `ViewModel` classes for sample pages in the code-behind of the control as I have it like this for all storage samples. Unfortunately I probably did some additional changes to these samples in the subsequent branches, so I would prefer to keep it this way for now so that I don't introduce more conflicts when I rebase.